src/Pages/SearchPages/BlockPanel.js

```diff
@@ -11,66 +11,73 @@ export default class BlockPanel extends Component {
         var block_link = "/search/" + data.hash;
         var miner_link = "/search/" + miner;
         return (
-            <Container fluid style={{"paddingLeft": "0px", "paddingRight": "0px", "height": "80px"}}>
-                <Table>
+                <Table responsive>
```

Comment:
![image](https://github.com/user-attachments/assets/dda02b25-70c9-4852-b48e-ce555372d70f)

If search for a block, icons Link to reproduce: /search/bfc1e0b3808670acb4c117a33282c99ae97e154a34e879c13bdbd441e68ac991